internal/sign/job/manager.go

```diff
@@ -15,46 +16,46 @@ import (
 	"github.com/kubernetes-sigs/kernel-module-management/internal/utils"
 )
 
-type signJobManager struct {
+type signPodManager struct {
 	client    client.Client
 	signer    Signer
-	jobHelper utils.JobHelper
+	podHelper utils.PodHelper
 	registry  registry.Registry
 }
 
-func NewSignJobManager(
+func NewSignPodManager(
 	client client.Client,
 	signer Signer,
-	jobHelper utils.JobHelper,
-	registry registry.Registry) *signJobManager {
-	return &signJobManager{
+	podHelper utils.PodHelper,
+	registry registry.Registry) *signPodManager {
+	return &signPodManager{
 		client:    client,
 		signer:    signer,
-		jobHelper: jobHelper,
+		podHelper: podHelper,
 		registry:  registry,
 	}
 }
 
-func (jbm *signJobManager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
-	jobs, err := jbm.jobHelper.GetModuleJobs(ctx, modName, namespace, utils.JobTypeSign, owner)
+func (jbm *signPodManager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
```

Comment:
```suggestion
func (spm *signPodManager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
```
Please rename all receivers in this file.

Reply:
Thanks. Fixed.